Workshop4- Homework/myworld/myworld/celery.py

```diff
@@ -0,0 +1,33 @@
+import os
+from celery import Celery
+
+# Set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'myworld.settings')
+
+app = Celery('myworld')
+
```

Comment:
@Rahul-H-Raj can you give me usecases of celery?

Reply:
@anushds  Some of the use cases of Celery are:
- Task scheduling and execution: Celery can be used to schedule tasks to run at specific times or on recurring schedules. This is useful for automating periodic tasks like data backups, database maintenance, and regular data updates
- Parallel Processing: It can be used to distribute tasks across multiple worker nodes so that it is beneficial for tasks that can be parallelized, such as web scraping, data processing, and rendering
- Scalability for handling large workloads: Celery makes it easier to scale by adding more worker nodes to handle increasing tasks.
- Performing long-running tasks: Celery can be used to perform long-running tasks, such as processing large datasets or generating reports, in the background.
- Real-time Data Processing: Celery can be used for real-time data processing, such as handling live data streams, event processing, and real-time analytics. It can process data as it arrives, making it valuable in scenarios like monitoring and reporting.